state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -505,6 +508,14 @@ pub mod pallet {
 		ConstU32<DEFAULT_MAX_SWAP_REQUEST_DURATION_BLOCKS>,
 	>;
 
+	/// The minimum chunk size for DCA swaps. The number of chunks of a DCA swap will be reduced
+	/// so that the chunk size is greater than or equal to this value. Setting to zero will disable
+	/// the check for that asset.
+	#[pallet::storage]
+	#[pallet::getter(fn minimum_chunk_size)]
+	pub type MinimumChunkSize<T: Config> =
```

Comment:
The `MinimumChunkSize` will default to 0 for all assets, meaning that there will be no minimum, the check is skipped. 
If we want to set an actual default, I think it will require a migration. Should I make this migration or just let governance set it once its released? @dandanlen